frontend/src/containers/userGroup/UserGroupContainer.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useCallback } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import UserGroup from '../../components/UserGroup/UserGroup';
+import { withRouter } from 'react-router-dom';
+import { notifyDriverLocation, onTaxi } from '../../modules/group/group';
+
+// todo: need to send Google Map API as props
+// todo: need to send driver info as props
+// current state : group has been formed
+
+function UserGroupContainer() {
+  const dispatch = useDispatch();
+  const group = useSelector(group => ({
+    group: group.group,
+  }));
+
+  const onClickOnTaxi = useCallback(id => dispatch(onTaxi(id)), [dispatch]);
+  // group.driver is currently null
+  // so will use temporary driver info
+  const driverInfo = 'Name: mockDriver, Vehicle : BMW, Plate No. : 01A 1234';
```

Comment:
I think `driverInfo` is not string type but object

![image](https://user-images.githubusercontent.com/18715362/68134100-f42e5380-ff64-11e9-8f4c-cbefd88d15dd.png)

maybe driver info object would be similar to this.

change test driver info into object please